src/sysen1/netime.128

```diff
@@ -100,45 +100,80 @@ $forma:	save a
 
 hosts:
 
+; Ask the reliable machines first.  They are all on the global
+; Chaosnet and tend to answer quickly.
+
+	[asciz "LOCAL-TIME-SERVER"]
+	[asciz "TIME.CHAOSNET.NET"]
+	[asciz "CHAOS.SWENSON.ORG"]
+	[asciz "CHAOS2.SWENSON.ORG"]
+	[asciz "ROUTER.CHAOSNET.NET"]
```

Comment:
Remove ROUTER since it is the same machine as TIME.

Reply:
LOCAL-TIME-SERVER isn't in the H3TEXT file.  Maybe do it in #2226?

Reply:
Or add it in this commit?

Reply:
I stupidly already committed after seeing Lars’ approval and before getting through my email. I’ll add another PR removing router.chaosnet.org and adding local-time-server to h3text.